schema-lending-extended.graphql

```diff
@@ -1001,11 +969,25 @@ type Position @entity {
   dailySnapshots: [PositionSnapshot!]! @derivedFrom(field: "position")
 }
 
+# Unlike other snapshots that are taken at a fixed time interval. Position
+# snapshots should be taken after every event, including the opening and
+# closing events. This will prevent an ever growing number of snapshots
+# for positions that are not moving. As we are only recording the balance
+# in token amounts instead of in USD, this will work well.
 # Note that we only take snapshot for open positions
 type PositionSnapshot @entity {
```

Comment:
the entity doesn't come with a timestamp field, then how do we order them?

Reply:
which entity doesn't come with a timestamp field?

Reply:
my bad, saw it